package de.sj4.coronapartyapp.exception

class NoAuthCodeSetException : Exception("there is no valid auth code set in local storage")
